### ISSUE TITLE
No recipe in JEI for cristallized amber and obsidian, and exp confection + impossible to do cristallized obsidian

### ISSUE THREAD

**Issue by antoine-courant** [PrinceOfAmber/Cyclic#2084]
Minecraft Version: 1.18.2

Forge Version: 40.0.17

Mod Version: 1.6.18

Single Player or Server: Multiplayer on server

Describe problem (what you were doing; what happened; what should have happened):
I noticed that few recipes are missing on JEI, I can't find how to do cristallized amber and cristallized obsidian, neither the experience confection
I read on internet how to make amber and I was able to make it, but I can't make cristallized obsidian, I think it's either from top to bottom : obsidian, cristallized amber and wither rose (I tried in reverse order as well) or emerald, cristallized amber and chorus fruit, but none of them work
Also because of JEI not showing recipe, I don't know how to make exp confection
The cyclic guidebook doesn't say anything as well on how to make those 3 items :/

**Comment by HalcyonSeas73** [comment]
Are you using a modpack? Modpacks can disable certain recipes. The recipes are functional in 1.18.2 via JEI on a standalone single-player world.

**Comment by antoine-courant** [comment]
Ah ! I'm on Better Minecraft plus, 1.18.2 v2, so there's a lot of mods sadly :/

**Comment by HalcyonSeas73** [comment]
> Ah ! I'm on Better Minecraft plus, 1.18.2 v2, so there's a lot of mods sadly :/

Then that may be why; the modpack is probably disabling the recipes. Not a Cyclic issue.

**Comment by antoine-courant** [comment]
Pretty weird to be those two specifics and not other ones
Is there somewhere in the config file of cyclic where it could be disabled ? I can't find it in both files (server and client)

**Comment by HalcyonSeas73** [comment]
> Pretty weird to be those two specifics and not other ones Is there somewhere in the config file of cyclic where it could be disabled ? I can't find it in both files (server and client)

They're likely disabled for balance. It isn't in config files. You're using a modpack, therefore it would be better to contact the modpack authors about this issue.

**Comment by antoine-courant** [comment]
I see, so it's on purpose for the balance of modpack, thanks for the information ;)
To be sure, what are the recipes/conditions to meet to make crystallized obsidian and exp confection ?

**Comment by Lothrazar** [comment]
the default recipes can be seen with something like JEI , and it sounds like you would have to check with the modpack authors issue tracker for their recipes

**Comment by antoine-courant** [comment]
Well to be honest, this mod wasn't in the modpack originally, I added it myself, but I just put the jar in the mod folder, I didn't do anything else other than that
Isn't this disabled by default ?

**Comment by antoine-courant** [comment]
Yeah I checked again, I added Cyclic separately from the modpack, it wasn't in it originally, so I don't see where it could have been disabled

**Comment by HalcyonSeas73** [comment]
> Yeah I checked again, I added Cyclic separately from the modpack, it wasn't in it originally, so I don't see where it could have been disabled

Regardless, it isn't a Cyclic issue, it's otherwise clashing with the modpack. Cyclic is working as intended. Even added separately from the modpack.

**Comment by Floating8Tres** [comment]
I'm having the same issue with the crystallized obsidian. Recipe doesn't show in JEI nor the Cyclic guidebook (it's obsidian, crystalized amber, then chorus fruit with liquid slime; correct?). I've removed every mod and removed the datapacks in my world and the issue is still present. I started a world in creative and tried there to make crystallized obsidian but to no avail.
I'm also running 1.18.2 with Forge version 40.1.0 in singleplayer, and mod version 1.7.2

I don't know if I should raise the issue in a different thread or just post it here and the issue will be reopened :\
